client/state/selectors/get-new-site-coming-soon-setting.ts

```diff
@@ -12,3 +12,7 @@ import shouldNewSiteBePrivateByDefault from './should-new-site-be-private-by-def
 export default function getNewSiteComingSoonSetting( state: object ): number {
 	return shouldNewSiteBePrivateByDefault( state ) ? 1 : 0;
 }
+
+export function getNewSiteComingSoonSettingV2( state: object ): number {
+	return shouldNewSiteBePrivateByDefault( state ) ? 0 : 1;
```

Comment:
I'm reluctant to use `shouldNewSiteBePrivateByDefault` at all inside this function. That's because coming-soon-v2 has nothing to do with private-by-default, and the future of private-by-default affect v2 in any way. In fact v2 is an attempt to try to disentangle coming-soon from privacy. 

I know it doesn't make any difference to the actual outcome (since `shouldNewSiteBePrivateByDefault()` is hard-coded to return `true`) but I think it will further tie the two things together in peoples minds.

Reply:
I changed the logic here so that I was able to create a coming-soon-v2 site using the `/start` flow. The above logic was basically hard-coded to always return 0.

Reply:
This was a bit of hand wringing by me. I was trying to keep the code sort of similar to v1 but I think it made things way more complicated, I've taken your change and gone further by dropping the function completely and just using the feature flag directly. I think its still a bit confusing but if we do a sweep with your visibility enum we can probably drop the one line functions that just return constant values completely anyway.